### ISSUE TITLE
Accuracy lower than reported

### ISSUE THREAD

**Issue by fil-mp** [kekmodel/FixMatch-pytorch#27]
For the 40 labels on CIFAR-10 the accuracy reaches 93.38, when I run it. I used the same hyperparameters and seed 5. Is the reported acc for seed 5?

**Comment by kekmodel** [comment]
Yes. I trained using `--seed 5`.

**Comment by fil-mp** [comment]
OK. Thank you.
